tsconfig.json

```diff
@@ -22,77 +22,6 @@
   },
   "include": ["./src/**/*", "next-env.d.ts", "**/*.ts", "**/*.tsx"],
   "exclude": [
-    "node_modules",
-    "./public/content",
-    "./src/intl",
-    "./src/data", // TODO: move to /public?
```

Comment:
I still think it is worth keeping these paths because they reduce the amount of file scanning that TS has to do.

From the [ts docs](https://www.typescriptlang.org/tsconfig#exclude):
> Important: exclude only changes which files are included as a result of the include setting. A file specified by exclude can still become part of your codebase due to an import statement in your code, a types inclusion, a /// <reference directive, or being specified in the files list.
> It is not a mechanism that prevents a file from being included in the codebase - it simply changes what the include setting finds.

Sidenote, I'd also modify the `"./public/content"` to exclude the entire `"./public"` folder.

Reply:
Agree... My comment above was specific to `/data` which has TS files we should be checking